howto/troubleshoot/troubleshoot-lxd-cluster

```diff
@@ -0,0 +1,66 @@
+The following issues might occur if you use clustering.
+
+<a name="ams-lxd-relation-changed-failed"></a>
```

Comment:
Should this be a markdown link instead? Or maybe we can completely remove it?

Reply:
This is actually a hack used for cross-file anchoring. However, I am removing this because this particular anchor is not used anywhere else. You may see other similar anchors which are added so that we can redirect to a specific section of a topic.